src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx

```diff
@@ -202,13 +198,6 @@ function CollectionEditor({ viewerRef }: Props) {
 
   const shouldDisplayEditor = stagedNfts.length > 0;
 
-  const { showModal } = useModalActions();
-  const user = viewer.user;
-
-  useKeyDown('Escape', () => {
-    showModal({ content: <ConfirmLeaveModal userRef={user} /> });
-  });
```

Comment:
We no longer need to handle any escape keypresses in the `CollectionEditor` component